specification/servicefabricmanagedclusters/resource-manager/Microsoft.ServiceFabric/preview/2023-07-01-preview/managedcluster.json

```diff
@@ -0,0 +1,2192 @@
+        "description": "Action to get Az Resiliency Status of all the Base resources constituting Service Fabric Managed Clusters.",
+        "operationId": "managedAzResiliencyStatus_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          }
+        ],
+        "x-ms-examples": {
+          "Az Resiliency status of Base Resources": {
+            "$ref": "./examples/managedAzResiliencyStatusGet_example.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "The operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/ManagedAzResiliencyStatus"
+            }
+          },
+          "default": {
+            "description": "The detailed error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorModel"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceFabric/managedClusters/{clusterName}/getmaintenancewindowstatus": {
+      "post": {
+        "tags": [
+          "managedMaintenanceWindowStatus"
+        ],
+        "summary": "Action to get Maintenance Window Status of the Service Fabric Managed Clusters.",
+        "description": "Action to get Maintenance Window Status of the Service Fabric Managed Clusters.",
+        "operationId": "managedMaintenanceWindowStatus_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          }
+        ],
+        "x-ms-examples": {
+          "Maintenance Window Status": {
+            "$ref": "./examples/ManagedMaintenanceWindowStatusGet_example.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "The operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/ManagedMaintenanceWindowStatus"
+            }
+          },
+          "default": {
+            "description": "The detailed error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorModel"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceFabric/managedClusters/{clusterName}/applymaintenancewindow": {
```

Comment:
>applymaintenancewindow [](http://example.com/codeflow?start=136&length=22)

This should originally have been camelCased. Do breaking changes folks recommend fixing it, or leaving it as is? /cc @mikekistler